life_expectancy/cleaning.py

```diff
@@ -0,0 +1,97 @@
+from pathlib import Path
+import argparse
+import pandas as pd
+
+
+CURRENT_DIR = str(Path(__file__).parent)
+LIFE_EXPECTANCY_DATA_PATH = CURRENT_DIR + "/data/eu_life_expectancy_raw.tsv"
+OUTPUT_DATA_PATH = CURRENT_DIR + "/data/pt_life_expectancy.csv"
```

Comment:
If you filter for a region that is not 'PT', the output will still be saved as 'pt_life_expectancy.csv". I would suggest making the name of the outputted *.csv file dynamic (in accordance with the selected region).

Reply:
Good catch! I solved it in this commit https://github.com/2010b9/learning-pod-foundations-assignments/pull/5/commits/e40b93399e9878f75159fdc36b485cdf57958427 🙂